pkg/config.go

```diff
@@ -19,8 +19,14 @@ type BaseConfig struct {
 
 type RDSConfig struct {
 	BaseConfig `yaml:"base,inline"`
-	Regions    []string  `yaml:"regions"`
-	EOLInfos   []EOLInfo `yaml:"eol_info"`
+	Regions    []string    `yaml:"regions"`
+	EOLInfos   []EOLInfo   `yaml:"eol_info"`
+	Thresholds []Threshold `yaml:"thresholds"`
+}
+
+type Threshold struct {
+	Name      string `yaml:"name"`
+	Threshold int    `yaml:"threshold"`
```

Comment:
Threshold.Threshold? That's confusing :), perhaps Value?

Reply:
Thresholds makes sense to me. It is the point at which the EOL status will change. In the configmap it will look like:
      thresholds:
        - name: green
          threshold: 365
        - name: yellow
          threshold: 180
        - name: red
          threshold: 90

Reply:
Nevermind, I see what you mean :)

Reply:
How about Thresholds.Threshold ?

Reply:
In the config `thresholds` is a list of `Threshold`, each `Threshold` contains 2 fields (name and int), to make it clear that int means day, how about change it to

```yaml
thresholds:
- name: green
  days: 365
```

`Threshold.Days`